adafruit_featherwing/keyboard_featherwing.py

```diff
@@ -48,31 +48,43 @@
 
 
 # pylint: disable-msg=too-few-public-methods
+# pylint: disable-msg=too-many-arguments
 class KeyboardFeatherwing:
     """Class representing a `Keyboard Featherwing`
        <https://www.tindie.com/products/arturo182/keyboard-featherwing-qwerty-keyboard-26-lcd/>`_.
 
        """
 
-    def __init__(self, spi=None, cs=None, dc=None, i2c=None):
+    def __init__(
+        self,
+        spi=None,
+        cs=None,
+        dc=None,
+        i2c=None,
+        ts_cs=None,
+        sd_cs=None,
+        neopixel_pin=None,
+    ):
         displayio.release_displays()
         if spi is None:
             spi = board.SPI()
         if cs is None:
             cs = board.D9
-        if dc is None:
-            dc = board.D10
```

Comment:
was the default for `dc` here removed intentionally?

Reply:
No -- I will restore it -- surprised it worked OK on the stm32...

Reply:
restored - thanks for catching that.